domains/vikram.json

```diff
@@ -7,6 +7,6 @@
     "twitter": "vikram_v18"
   },
   "record": {
-    "CNAME": "vikramisdev.github.io"
+    "A": "165.2.6.8"
```

Comment:
A records should be an array of ips
also I cannot reach your website, keeps on loading

Reply:
its vikramisdev.netlify.app, and i am confused in connecting this site to this domain, i want to remove the older github pages site with this, how can i do it.